src/main/java/org/scijava/script/ScriptFinder.java

```diff
@@ -178,8 +179,10 @@ private int createInfos(final List<ScriptInfo> scripts, final Set<URL> urls,
 			// friendlyPath = "File/Import/Movie File..."
 			// menuPath = File > Import > Movie File...
 
-			// NB: Ignore base-level scripts (not nested in any menu).
-			if (menuPath.size() == 1) continue;
+			// Place base-level scripts in the "Scripts" submenu
+			if (menuPath.size() == 1){
+				menuPath.add(0, new MenuEntry("Scripts"));
```

Comment:
We don't want a new top-level "Scripts" directory. I suggest "Plugins > Scripts" since that matches their location on disk.